analyses/molecular-subtyping-HGG/03-HGG-molecular-subtyping-cnv.Rmd

```diff
@@ -48,6 +43,74 @@ if (!dir.exists(results_dir)) {
 
 ### Read in Files
 
+We will need the clinical/histologies file and the defining lesions file to distinguish between missing samples or samples that had only neutral copy number calls.
+
+```{r message=FALSE}
+histologies_df <- read_tsv(file.path(root_dir, "data",
+                                     "pbta-histologies.tsv"))
+defining_lesions_df <- read_tsv(file.path(results_dir, 
+                                          "HGG_defining_lesions.tsv"))
+```
+
+#### Inclusion Criteria
+
+Here, we'll use the same logic as `02-HGG-molecular-subtyping-subset-files.R` to identify samples to include and also check for experimental strategy.
+
+```{r}
+hgg_lesions_df <- defining_lesions_df %>%
+  dplyr::filter(
+    short_histology == "HGAT" |
+      grepl("H3 G35 mutant|H3 K28 mutant", disease_type_reclassified)
+  )
+
+# Filter metadata for HGAT and samples that should be classified
+# as High-grade glioma based on defining lesions
+hgg_metadata_df <- histologies_df %>%
+  dplyr::filter(
+    short_histology == "HGAT" |
+      sample_id %in% hgg_lesions_df$sample_id,
+    sample_type == "Tumor",
+    composition == "Solid Tissue"
+  )
+```
+
+Only WGS samples have CNV and SV data, so we'll filter based on `experimental_strategy`.
+
+```{r}
+included_wgs_samples <- intersect(hgg_lesions_df %>% 
+                                    pull(Kids_First_Biospecimen_ID),
+                                  hgg_metadata_df %>% 
+                                    filter(experimental_strategy == "WGS") %>%
+                                    pull(Kids_First_Biospecimen_ID))
+```
+
+Are all the WGS samples that pass the inclusion criteria in the consensus SEG file committed to the repository?
+
+```{r}
+consensus_seg_ids <- read_tsv(file.path(root_dir, 
+                                        "analyses", 
+                                        "copy_number_consensus_call",
+                                        "results",
+                                        "pbta-cnv-consensus.seg.gz")) %>%
+  pull(ID)
+consensus_seg_ids <- unique(consensus_seg_ids)
+```
+
+```{r}
+all(included_wgs_samples %in% consensus_seg_ids)
+```
+
+No, they're not.
+How many are missing?
+
+```{r}
+sum(!(included_wgs_samples %in% consensus_seg_ids))
+```
+
+We'd then expect `r sum(included_wgs_samples %in% consensus_seg_ids)` samples to have non-NA values in the cleaned CNV files.
```

Comment:
These `NA`s are separate than those that eventually got switched to be called "wildtype" right? Just trying to follow what ended up happening to these samples and how it fits in with the final table.

Reply:
This is copy number data and the calls are ultimately based on SNV data.